NumberBaseball/NumberBaseball.c

```diff
@@ -23,81 +33,85 @@ void Init(int* number1, int* number2, int* number3)
 			continue;
 		}
 
-		*number1 = test1;
-		*number2 = test2;
-		*number3 = test3;
+		computer->guess1 = test1;
+		computer->guess2 = test2;
+		computer->guess3 = test3;
 		break;
 	}
 }
 
-void Update(int* number1, int* number2, int* number3, int* guess1, int* guess2, int* guess3, int* entry)
+void Update(guess* player, int* entry)
 {
 	while (*entry == 1)
 	{
-		char line[LINESIZE];
-		int numbersEntered = 0;
-		*guess1 = -1;
-		*guess2 = -1;
-		*guess3 = -1;
-
-		if (!fgets(line, LINESIZE, stdin))
+		char line[LINE_SIZE];
+		size_t test1 = -1;
+		size_t test2 = -1;
+		size_t test3 = -1;
+		size_t* guess1 = &test1;
+		size_t* guess2 = &test2;
+		size_t* guess3 = &test3;
+
+		if (!fgets(line, LINE_SIZE, stdin))
 		{
 			clearerr(stdin);
-			printf("\n");
-			exit(1);
+			printf("Error reading numbers please try again or restart program.\n");
+			continue;
 		}
 
-		if ((numbersEntered = sscanf_s(line, "%d %d %d",
-			guess1, guess2, guess3)) == 3)
+		if ((sscanf_s(line, "%u %u %u", guess1, guess2, guess3)) == 3)
 		{
+			player->guess1 = test1;
+			player->guess2 = test2;
+			player->guess3 = test3;
 			return;
 		}
-		else
-		{
-			*entry = -1;
-			return;
-		}
+
+		*entry = -1;
+		return;
 	}
 }
 
-int Render(int* number1, int* number2, int* number3, int* guess1, int* guess2, int* guess3, int* entry)
+int Render(guess* computer, guess* player, int* entry)
 {
 	if (*entry == 0)
 	{
 		printf("Welcome to Number Baseball!\n");
-		printf("Choose 3 numbers:\n");
+		printf("Please choose 3 unique numbers from 0 to 9\n");
 		*entry = 1;
 	}
 	else if (*entry == -1)
 	{
-		printf("Invalid amount of numbers. Please choose 3 numbers.\n");
+		printf("Invalid amount of numbers.\n Please choose 3 unique numbers from 0 to 9.\n");
 		*entry = 1;
 	}
 	else
 	{
-		int correctCount = 0;
-		int* guess[] = {guess1, guess2, guess3};
-		int* number[] = {number1, number2, number3, number1, number2};
+		size_t strikeCount = 0;
+		size_t outCount = 0;
+		size_t ballCount = 0;
+		size_t* guess[] = {player->guess1, player->guess2, player->guess3};
```

Comment:
This is wrong..you are storing size_t variables into an array of pointer to size_t.
Should be an array of size_t